gunicorn/reloader.py

```diff
@@ -26,7 +28,7 @@ def add_extra_file(self, filename):
 
     def get_files(self):
         fnames = [
-            re.sub('py[co]$', 'py', module.__file__)
+            COMPILED_EXT_RE.sub('py', module.__file__)
             for module in list(sys.modules.values())
```

Comment:
If we want to optimize this further, we can remove `list()` calls around `sys.modules.values()` as well. `sys.modules.values()` returns a plain `list` in Python 2 and `dict_values` view in Python 3 (even better!) so no need create another list object.

Reply:
I agree. Looked unnecessary regardless of speed. Will add it in.